roles/logging/tasks/filebeat.yml

```diff
@@ -55,6 +55,9 @@
     - refresh cert auths
     - restart filebeat
   when: logging.forward.host and logging.forward.tls.enabled
+  
+- include: checks.yml
```

Comment:
to be more consistent we should change this to be:

```
- include: monitoring.yml
  tags: [monitoring, sensu-checks]
when: monitoring.enabled|default('True')|bool
```

we should probably also move it to main.yml and remove it from filebeat and logstash-forwarder plays since both should run it.